validator/src/main/java/com/amazon/aoc/helpers/CWMetricHelper.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.aoc.helpers;
+
+import com.amazon.aoc.callers.ICaller;
+import com.amazon.aoc.fileconfigs.FileConfig;
+import com.amazon.aoc.models.Context;
+import com.amazonaws.services.cloudwatch.model.Dimension;
+import com.amazonaws.services.cloudwatch.model.Metric;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * helper class for getting expected metrics from templates.
+ */
+public class CWMetricHelper {
```

Comment:
Is it possible to make the functions in Helper class static?

Reply:
i wonder what's the benefit to make it static?  I tended to use non-static class/function as they are more friendly to unit-test, but i'm open to change it if it's the best practice to use static here.

Reply:
Usually `static` could better explain that methods are stateless, it would also slightly improve the runtime performance, which won't make any difference here in the integration test... I don't think it worths much effort to change it to static.

Reply:
thanks! learned now :)